test/testdrive/mz-support-privileges.td

```diff
@@ -44,3 +44,11 @@ EXPLAIN TIMESTAMP FOR SELECT b.auction_id, b.buyer, b.amount, b.bid_time, u.org_
 # The mz_support user can filter SHOW commands on user objects.
 $ postgres-execute connection=mz_support
 SHOW INDEXES ON bids;
+
+# The mz_support user can SHOW public system vars.
+$ postgres-execute connection=mz_support
+SHOW max_tables;
+
+# The mz_support user can SHOW internal system vars.
+$ postgres-execute connection=mz_support
+SHOW log_filter;
```

Comment:
Can you add a test to make sure that `mz_support` cannot alter these values?

Reply:
I think I cannot do that with the current `*.td` syntax. There is no way to both run under a different connection (`$ postgres-execute`) and at the same time assert that an error is met (`!` + `contains:`).

Reply:
Oh right, oh well, that's fine.

Reply:
I think I might be able to do this in an `*.slt` test.